packages/template/src/components-page/account-settings.tsx

```diff
@@ -357,6 +367,182 @@ function EmailsAndAuthPage() {
+        .map(session => user.revokeSession(session.id));
+      await Promise.all(deletionPromises);
+      setSessions(sessions.filter(session => session.is_current));
+    } catch (error) {
+      console.error('Failed to revoke all sessions:', error);
+    } finally {
+      setIsRevokingAll(false);
+      setShowConfirmRevokeAll(false);
+    }
+  };
+
+  return (
+    <PageLayout>
+      <div>
+        <div className="flex justify-between items-center mb-2">
+          <Typography className='font-medium'>{t("Active Sessions")}</Typography>
+          {sessions.filter(s => !s.is_current).length > 0 && !isLoading && (
+            showConfirmRevokeAll ? (
+              <div className="flex gap-2">
+                <Button
+                  variant="destructive"
+                  size="sm"
+                  loading={isRevokingAll}
+                  onClick={handleRevokeAllSessions}
+                >
+                  {t("Confirm")}
+                </Button>
+                <Button
+                  variant="secondary"
+                  size="sm"
+                  disabled={isRevokingAll}
+                  onClick={() => setShowConfirmRevokeAll(false)}
+                >
+                  {t("Cancel")}
+                </Button>
+              </div>
+            ) : (
+              <Button
+                variant="outline"
+                size="sm"
+                onClick={() => setShowConfirmRevokeAll(true)}
+              >
+                {t("Revoke All Other Sessions")}
+              </Button>
+            )
+          )}
+        </div>
+        <Typography variant='secondary' type='footnote' className="mb-4">
+          {t("These are devices where you're currently logged in. You can revoke access to end a session.")}
+        </Typography>
+
+        {isLoading ? (
+          <Skeleton className="h-[300px] w-full rounded-md" />
+        ) : (
+          <div className='border rounded-md'>
+            <Table>
+              <TableHeader>
+                <TableRow>
+                  <TableHead className="w-[200px]">{t("Session")}</TableHead>
+                  <TableHead className="w-[150px]">{t("IP Address")}</TableHead>
+                  <TableHead className="w-[150px]">{t("Location")}</TableHead>
+                  <TableHead className="w-[150px]">{t("Last used")}</TableHead>
+                  <TableHead className="w-[80px]"></TableHead>
+                </TableRow>
+              </TableHeader>
+              <TableBody>
+                {sessions.length === 0 ? (
+                  <TableRow>
+                    <TableCell colSpan={5} className="text-center py-6">
+                      <Typography variant="secondary">{t("No active sessions found")}</Typography>
+                    </TableCell>
+                  </TableRow>
+                ) : (
+                  sessions.map((session) => (
+                    <TableRow key={session.id} className={session.is_current ? "bg-accent/10" : ""}>
```

Comment:
```suggestion
                    <TableRow key={session.id} className={cn({
                      'bg-accent/10': session.is_current,
                    })}>
```

use `cn` to format classnames